src/config.ts

```diff
@@ -1,3 +1,12 @@
+import { resolve } from "path";
+import { config } from "dotenv"
+
+config({ path: resolve(__dirname, "../.env") })
+
+export const BACKEND_URL = process.env.BACKEND_URL || 'http://localhost';
+export const FRONTEND_URL = process.env.FRONTEND_URL || 'http://localhost';
+export const BACKEND_PORT = parseInt(process.env.BACKEND_PORT || "5000");
```

Comment:
what's the difference between server_port and backend_port?

Reply:
Legacy.
I left it for backward compatibility only since people are sometimes using it and sometimes don't and we end up with hardcoded urls 'localhost:5000' and ports all over the place.
Refactoring is needed though.